staff/judith-gongora/js-arrays/slice.js

```diff
@@ -0,0 +1,64 @@
+// function slice (arr, num, num2){
+//     if(num2==undefined){
+//         num2=arr.length
+//     }
+//     var e= 0;
+//     var array=[]
+//     for (i=num;i<num2;i++){
+//         array[e]=arr[i];
+//         e++;
+//     }
+     
+//     return array;
+// }
+
+function slice(arr, start, end) {
```

Comment:
I tried this in console and it didn't work?

Reply:
My code is the comment